server/eclipse-project/src/main/java/au/org/democracydevelopers/corla/model/vote/IRVChoices.java

```diff
@@ -112,9 +120,34 @@ public IRVChoices(List<IRVPreference> mutableChoices) {
    * @throws IRVParsingException if one of the comma-separated substrings cannot be parsed as a
    *                             name(rank). This could happen for example if called on a
    *                             plurality vote.
+   * TODO Note we are currently not using this outside tests - it's possible the List<String>
+   *   constructor is the only one we need.
    */
   public IRVChoices(String sanitizedChoices) throws IRVParsingException {
-    this(parseChoices(sanitizedChoices));
+    this(Arrays.asList(sanitizedChoices.trim().split(",")));
+  }
+
```

Comment:
Check the formatting of the rawChoices parameter description -- it looks like a new line is started for "repeated or skipped ...".